src/reducers/tweets.js

```diff
@@ -0,0 +1,13 @@
+import { TWEET_RECEIVED } from '../actions';
+
+export default function (state = [], action) {
+  switch (action.type) {
+  case TWEET_RECEIVED:
+    return [
+      ...state.slice(-99),
+      action.payload
+    ];
+  default:
+    return state;
+  }
+}
```

Comment:
Hm, jeg tenker det sånn at "Hva er payload når man mottar en tweet? En tweet". Samme med den andre action-en (`TWEET_SELECTED` - "hva kan payload være? en tweet."). Å ha `payload` som et objekt med `tweet` som en property føler jeg blir worst of both worlds.

Når det er sagt så kan vi droppe hele `payload`-greia og la det være `action.tweet` i begge tilfeller, hvis du/dere syns det er bedre :)